grails-forge-core/src/main/java/org/grails/forge/feature/github/workflows/plain/templates/plainGithubWorkflow.rocker.raw

```diff
@@ -12,23 +12,24 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      checks: write
+      contents: read
     steps:
 @javaSetup.template(jdkVersion)
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@@v3
```

Comment:
The `gradle/actions/setup-gradle` action replaces `gradle/gradle-build-action`.
Reference:
https://github.com/gradle/actions/tree/af6e5767241073b8ab20cc8cf7e95ec63fadc160/setup-gradle#the-setup-gradle-action